R/collapse_duplicates.R

```diff
@@ -149,3 +149,33 @@ findDuplicates <- function (db,
         arrange(finddups_row_idx) %>%
         select(!any_of(c('collapse_idx', 'finddups_row_idx')))
 }
+
+#' This is the function to truncate a single sequence at a specific position from 5' end
+#' @export
+truncate_from_5_end <- function(sequence_aligned, truncate_position_5_end) {
```

Comment:
```suggestion
mask_5_prime_seq <- function(sequence_aligned, imgt_position) {
```

Reply:
I would suggest to call the function "mask 5 prime" as we don't actually cut or truncate the sequence but mask it with "N"

Reply:
Also I would call the position "IMGT" position as it is done according to the IMGT numbering